apps/aeutil/lib/serialization.ex

```diff
@@ -128,26 +126,26 @@ defmodule Aeutil.Serialization do
 
   def serialize_value(value, type) when is_binary(value) do
     case type do
+      :root_hash ->
+        ChainState.base58c_encode(value)
+
       :prev_hash ->
-        Header.bech32_encode(value)
+        Header.base58c_encode(value)
 
       :txs_hash ->
-        SignedTx.bech32_encode_root(value)
-
-      :chain_state_hash ->
-        ChainState.bech32_encode(value)
+        SignedTx.base58c_encode_root(value)
 
       :from_acc ->
-        Aewallet.Encoding.encode(value, :ae)
+        Account.base58c_encode(value)
 
       :to_acc ->
-        Aewallet.Encoding.encode(value, :ae)
+        Account.base58c_encode(value)
 
       :signature ->
         base64_binary(value, :serialize)
 
       :proof ->
-        Aewallet.Encoding.encode(value, :ae)
+        Account.base58c_encode(value)
```

Comment:
I still don't get why this should be correct

Reply:
This is how the `proof` was encoded before. If there is a another prefix for the `proof` encoding we will change it.

Reply:
And I commented on that before, lets use base64 without prefix for now